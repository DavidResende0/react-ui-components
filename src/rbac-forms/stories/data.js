export const groups = [
  {
    value: '1',
    label: 'Hope',
  },
  {
    value: '2',
    label: 'Christa',
  },
  {
    value: '3',
    label: 'Tania',
  },
  {
    value: '4',
    label: 'Mcintyre',
  },
  {
    value: '5',
    label: 'Tonya',
  },
  {
    value: '6',
    label: 'Marcie',
  },
  {
    value: '7',
    label: 'Cara',
  },
  {
    value: '8',
    label: 'Becker',
  },
  {
    value: '9',
    label: 'Foley',
  },
  {
    value: '10',
    label: 'Perkins',
  },
];

export const usersTableColumns = [{
  property: 'fullname',
  label: 'Full Name',
}, {
  property: 'username',
  label: 'Username',
}, {
  property: 'email',
  label: 'E-mail',
}, {
  property: 'currentgroup',
  label: 'Current Group',
}, {
  property: 'role',
  label: 'Role',
}, {
  property: 'lastlogon',
  label: 'Last Logon',
}, {
  property: 'lastlogoff',
  label: 'Last Logoff',
}];

export const usersTableRows = [
  {
    id: 100,
    fullname: 'Adam the first',
    username: 'Admin',
    email: 'email@neco.com',
    currentgroup: 'Some group',
    role: 'Admin role',
    lastlogon: '06/25/18 06:58:14 UTC',
    lastlogoff: '06/30/18 06:58:14 UTC',
  },
  {
    id: 101,
    fullname: 'Marge the middle',
    username: 'La transparenza della qualita',
    email: 'email@jandak.com',
    currentgroup: 'Mediocre group',
    role: 'Shot role',
    lastlogon: '01/25/18 06:58:14 UTC',
    lastlogoff: '06/30/18 06:58:14 UTC',
  },
  {
    id: 102,
    fullname: 'Zlatko the last',
    username: 'Peligro',
    email: 'email@jandak.com',
    currentgroup: 'Super group',
    role: 'Shot role',
    lastlogon: '08/25/18 06:58:14 UTC',
    lastlogoff: '010/30/18 06:58:14 UTC',
  },
];

export const tags = [{
  name: 'Tag 1',
  value: 'Tag value 1',
}, {
  name: 'Tag 2',
  value: 'Tag value 2',
}, {
  name: 'Tag 3',
  value: 'Tag value 3',
}, {
  name: 'Tag 4',
  value: 'Tag value 4',
}, {
  name: 'Tag 5',
  value: 'Tag value 5',
}];
