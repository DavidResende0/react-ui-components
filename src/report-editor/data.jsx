/* Columns Tab - Available Fields */
export const columnOptions = [{
  value: 'Cloud Manager: Subscription',
  label: 'Cloud Manager: Subscription',
}, {
  value: 'Cloud Manager: Total Cloud Memory',
  label: 'Cloud Manager: Total Cloud Memory',
  type: 'number',
}, {
  value: 'Vms: Date Created',
  label: 'Vms: Date Created',
  type: 'datetime',
}, {
  value: 'Cloud Manager: Last Inventory Date',
  label: 'Cloud Manager: Last Inventory Date',
  type: 'datetime',
}, {
  value: 'Cloud Manager: Last Refresh Date',
  label: 'Cloud Manager: Last Refresh Date',
  type: 'datetime',
}, {
  value: 'Management Events: Date Created',
  label: 'Management Events: Date Created',
  type: 'datetime',
}, {
  value: 'Vms: Last Sync Time',
  label: 'Vms: Last Sync Time',
  type: 'datetime',
}, {
  value: 'CPU Usage Rate Average Avg Over Time Period',
  label: 'CPU Usage Rate Average Avg Over Time Period',
  type: 'number',
}, {
  value: 'Memory Usage Rate Average Avg Over Time Period',
  label: 'Memory Usage Rate Average Avg Over Time Period',
  type: 'number',
}, {
  value: 'Cloud Manager: Total Number of Logical CPUs',
  label: 'Cloud Manager: Total Number of Logical CPUs',
  type: 'number',
}, {
  value: 'Cloud Manager: Total Number of Physical CPUs',
  label: 'Cloud Manager: Total Number of Physical CPUs',
  type: 'number',
}, {
  value: 'Management Events: Container Group Name',
  label: 'Management Events: Container Group Name',
}, {
  value: 'Region Description',
  label: 'Region Description',
}, {
  value: 'Cloud Manager: Hostname',
  label: 'Cloud Manager: Hostname',
}, {
  value: 'Management Events: Destination VM Location',
  label: 'Management Events: Destination VM Location',
}];

/* Columns Tab - Configure Report Columns */
export const baseTheReportOptions = [
  {
    value: 'Availability Zones',
    label: 'Availability Zones',
  },
  {
    value: 'Bottleneck Events',
    label: 'Bottleneck Events',
  },
  {
    value: 'Chargeback for Images',
    label: 'Chargeback for Images',
  },
  {
    value: 'Chargeback for Projects',
    label: 'Chargeback for Projects',
  },
  {
    value: 'Chargeback for Vms',
    label: 'Chargeback for Vms',
  },
  {
    value: 'Cloud Networks',
    label: 'Cloud Networks',
  },
  {
    value: 'Cloud Providers',
    label: 'Cloud Providers',
  },
  {
    value: 'Cloud Resource Quotas',
    label: 'Cloud Resource Quotas',
  },
  {
    value: 'Chargeback for Vms',
    label: 'Chargeback for Vms',
  },
  {
    value: 'Cloud Tenants',
    label: 'Cloud Tenants',
  },
  {
    value: 'Cloud Volumes',
    label: 'Cloud Volumes',
  },
  {
    value: 'Cluster / Deployment Roles',
    label: 'Cluster / Deployment Roles',
  },
  {
    value: 'Compliance Histories',
    label: 'Compliance Histories',
  },
  {
    value: 'Configuration Managers',
    label: 'Configuration Managers',
  },
  {
    value: 'Configured Systems (Foreman)',
    label: 'Configured Systems (Foreman)',
  },
];

/* Columns Tab - Report Creation Timeout */
export const creationTimeoutOptions = [
  {
    value: '1 Hour',
    label: '1 Hour',
  },
  {
    value: '2 Hours',
    label: '2 Hours',
  },
  {
    value: '3 Hours',
    label: '3 Hours',
  },
  {
    value: '4 Hours',
    label: '4 Hours',
  },
  {
    value: '5 Hours',
    label: '5 Hours',
  },
  {
    value: '6 Hours',
    label: '6 Hours',
  },
];

/* Formatting Tab - PDF Output */
export const pdfOptions = [
  {
    value: 'A0 - 841mm x 1189mm',
    label: 'A0 - 841mm x 1189mm',
  },
  {
    value: 'A1 - 594mm x 841mm',
    label: 'A1 - 594mm x 841mm',
  },
  {
    value: 'A2 - 420mm x 594mm',
    label: 'A2 - 420mm x 594mm',
  },
  {
    value: 'A3 - 297mm x 420mm',
    label: 'A3 - 297mm x 420mm',
  },
  {
    value: 'A4 - 210mm x 297mm',
    label: 'A4 - 210mm x 297mm',
  },
  {
    value: 'US Executive - 7.25in x 10.5in',
    label: 'US Executive - 7.25in x 10.5in',
  },
  {
    value: 'US Folio - 8.5in x 13.0in',
    label: 'US Folio - 8.5in x 13.0in',
  },
  {
    value: 'US Government - 8.0in x 11.0in',
    label: 'US Government - 8.0in x 11.0in',
  },
  {
    value: 'US Executive - 17.0in x 11.0in',
    label: 'US Executive - 17.0in x 11.0in',
  },
  {
    value: 'US Legal - 8.5in x 14.0in',
    label: 'US Legal - 8.5in x 14.0in',
  },
];

/* Summary Tab - Sort Order */
export const sortOrderOptions = [
  {
    value: 'Asc',
    label: 'Ascending',
  },
  {
    value: 'Desc',
    label: 'Descending',
  },
];

/* Summary Tab - Sort Breaks */
export const sortBreaksOptions = [
  {
    value: 'No',
    label: 'No',
  },
  {
    value: 'Yes',
    label: 'Yes',
  },
  {
    value: 'Counts',
    label: 'Counts',
  },
];

/* Summary Tab - Number of rows to show */
export const rowNumberOptions = [
  {
    value: 'All',
    label: 'All',
  },
];

/* Summary Tab - Format on Summary Row */
export const rowOptions = [
  {
    value: 'Reset to default',
    label: 'Reset to default',
  },
  {
    value: 'Convert Numbers Larger than 1.0e+15 to Exponential Form',
    label: 'Convert Numbers Larger than 1.0e+15 to Exponential Form',
  },
  {
    value: 'Currency, 2 Decimals ($1,234.00)',
    label: 'Currency, 2 Decimals ($1,234.00)',
  },
  {
    value: 'Number (1,234)',
    label: 'Number (1,234)',
  },
  {
    value: 'Number (1,234.0)',
    label: 'Number (1,234.0)',
  },
];

/* Summary - Time Format on Summary Row */
export const formatSummaryTimeOptions = [
  {
    value: 'Date/Time (M/D/Y H:M:S Z)',
    label: 'Date/Time (M/D/Y H:M:S Z)',
  }, {
    value: 'Date (M/D)',
    label: 'Date (M/D)',
  }, {
    value: 'Date (M/D/YY)',
    label: 'Date (M/D/YY)',
  }, {
    value: 'Date (M/D/YYY)',
    label: 'Date (M/D/YYY)',
  }, {
    value: 'Day Full (Monday)',
    label: 'Day Full (Monday)',
  }, {
    value: 'Day Short (Mon)',
    label: 'Day Short (Mon)',
  }, {
    value: 'Day of Month (27)',
    label: 'Day of Month (27)',
  }, {
    value: 'Day of Month (27th)',
    label: 'Day of Month (27th)',
  }, {
    value: 'Day of Week (1)',
    label: 'Day of Week (1)',
  }, {
    value: 'Hour of Day (24)',
    label: 'Hour of Day (24)',
  }, {
    value: 'Month Full',
    label: 'Month Full',
  }, {
    value: 'Year (YYYY)',
    label: 'Year (YYYY)',
  },
];

/* Summary Tab - Original value to time fields */
export const timeValues = [
  {
    value: 'Original Value',
    label: 'Original Value',
  },
  {
    value: 'Hour',
    label: 'Hour',
  },
  {
    value: 'Day',
    label: 'Day',
  },
  {
    value: 'Week',
    label: 'Week',
  },
  {
    value: 'Month',
    label: 'Month',
  },
  {
    value: 'Quarter',
    label: 'Quarter',
  },
  {
    value: 'Year',
    label: 'Year',
  },
  {
    value: 'Hour of the Day',
    label: 'Hour of the Day',
  },
  {
    value: 'Day of the Week',
    label: 'Day of the Week',
  },
  {
    value: 'Day of the Month',
    label: 'Day of the Month',
  },
  {
    value: 'Week of the Year',
    label: 'Week of the Year',
  },
  {
    value: 'Month of the Year',
    label: 'Month of the Year',
  },
];

/* Summary Tab - Original value to time fields */
export const stringValues = [
  {
    value: 'Human Readable Model Name',
    label: 'Human Readable Model Name',
  }, {
    value: 'String Truncated to 50 Characters with Elipses (...)',
    label: 'String Truncated to 50 Characters with Elipses (...)',
  },
];

/* Consolidation - calculation */
export const calculationValues = [
  {
    value: 'Min',
    label: 'Min',
  }, {
    value: 'Max',
    label: 'Max',
  }, {
    value: 'Avg',
    label: 'Avg',
  }, {
    value: 'Total',
    label: 'Total',
  },
];

/* Formatting table - formats */
export const tableSelectOptions = {
  string: {
    defaultValue: 'Human Readable Model Name',
    options: [{
      value: 'Human Readable Model Name',
      label: 'Human Readable Model Name',
    }, {
      value: 'String Truncated to 50 Characters with Elipses (...)',
      label: 'String Truncated to 50 Characters with Elipses (...)',
    }],
  },
  datetime: {
    defaultValue: 'Date/Time (M/D/Y H:M:S Z)',
    options: [{
      value: 'Date/Time (M/D/Y H:M:S Z)',
      label: 'Date/Time (M/D/Y H:M:S Z)',
    }, {
      value: 'Date (M/D)',
      label: 'Date (M/D)',
    }, {
      value: 'Date (M/D/YY)',
      label: 'Date (M/D/YY)',
    }, {
      value: 'Date (M/D/YYY)',
      label: 'Date (M/D/YYY)',
    }, {
      value: 'Day Full (Monday)',
      label: 'Day Full (Monday)',
    }, {
      value: 'Day Short (Mon)',
      label: 'Day Short (Mon)',
    }, {
      value: 'Day of Month (27)',
      label: 'Day of Month (27)',
    }, {
      value: 'Day of Month (27th)',
      label: 'Day of Month (27th)',
    }, {
      value: 'Day of Week (1)',
      label: 'Day of Week (1)',
    }, {
      value: 'Hour of Day (24)',
      label: 'Hour of Day (24)',
    }, {
      value: 'Month Full',
      label: 'Month Full',
    }, {
      value: 'Year (YYYY)',
      label: 'Year (YYYY)',
    }],
  },
  number: {
    defaultValue: 'Number (1,234)',
    options: [{
      value: 'Number (1,234)',
      label: 'Number (1,234)',
    }, {
      value: 'Number (1,234.0)',
      label: 'Number (1,234.0)',
    }, {
      value: 'Number, 2 Decimals (1,234.00)',
      label: 'Number, 2 Decimals (1,234.00)',
    }, {
      value: 'Convert Numbers Larger than 1.0e+15 to Exponential Form',
      label: 'Convert Numbers Larger than 1.0e+15 to Exponential Form',
    }, {
      value: 'Currency, 2 Decimals ($1,234.00)',
      label: 'Currency, 2 Decimals ($1,234.00)',
    }],
  },
};
