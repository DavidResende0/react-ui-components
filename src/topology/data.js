export const nodes = [
  {
    id: 0,
    title: 'Levy',
    size: 25,
    fonticon: 'fa fa-cog',
  },
  {
    id: 1,
    title: 'Celina',
    size: 24,
    fonticon: 'fa fa-cog',
  },
  {
    id: 2,
    title: 'Nancy',
    size: 15,
    fonticon: 'fa fa-cog',
  },
  {
    id: 3,
    title: 'Yang',
    size: 25,
  },
  {
    id: 4,
    title: 'Gray',
    size: 28,
    fonticon: 'fa fa-cog',
  },
  {
    id: 5,
    title: 'Maddox',
    size: 16,
    fileicon: '/img/OpenShift-logo.svg',
  },
  {
    id: 6,
    title: 'Wallace',
    size: 27,
    utilization: 50,
    fonticon: 'fa fa-cog',
  },
  {
    id: 7,
    title: 'Bettie',
    size: 19,
    fonticon: 'fa fa-cog',
  },
  {
    id: 8,
    title: 'Watkins',
    size: 20,
    fonticon: 'fa fa-cog',
  },
  {
    id: 9,
    title: 'Stanton',
    size: 12,
    fileicon: '/img/OpenShift-logo.svg',
  },
  {
    id: 10,
    title: 'Lindsay',
    size: 18,
  },
  {
    id: 11,
    title: 'Harrell',
    size: 27,
  },
  {
    id: 12,
    title: 'Stephanie',
    size: 30,
    fileicon: '/img/OpenShift-logo.svg',
  },
  {
    id: 13,
    title: 'Mona',
    size: 18,
    fonticon: 'fa fa-cog',
  },
  {
    id: 14,
    title: 'Natalia',
    size: 21,
    fileicon: '/img/OpenShift-logo.svg',
  },
  {
    id: 15,
    title: 'Rose',
    size: 26,
    fonticon: 'fa fa-cog',
  },
  {
    id: 16,
    title: 'Robles',
    size: 12,
    fonticon: 'fa fa-cog',
  },
  {
    id: 17,
    title: 'Parker',
    size: 20,
    fonticon: 'fa fa-cog',
  },
  {
    id: 18,
    title: 'Decker',
    size: 21,
    fileicon: '/img/OpenShift-logo.svg',
  },
  {
    id: 19,
    title: 'Helen',
    size: 18,
  },
  {
    id: 20,
    title: 'Coleman',
    size: 30,
    fileicon: '/img/OpenShift-logo.svg',
  },
  {
    id: 21,
    title: 'Wolf',
    size: 25,
    fonticon: 'fa fa-cog',
  },
  {
    id: 22,
    title: 'Morton',
    size: 28,
    fileicon: '/img/OpenShift-logo.svg',
  },
  {
    id: 23,
    title: 'Fitzpatrick',
    size: 16,
    fileicon: '/img/OpenShift-logo.svg',
  },
  {
    id: 24,
    title: 'Hayden',
    size: 21,
  },
  {
    id: 25,
    title: 'Kaufman',
    size: 30,
    fileicon: '/img/OpenShift-logo.svg',
  },
  {
    id: 26,
    title: 'Julia',
    size: 15,
  },
  {
    id: 27,
    title: 'Louella',
    size: 16,
  },
  {
    id: 28,
    title: 'Rocha',
    size: 21,
  },
  {
    id: 29,
    title: 'Kimberly',
    size: 31,
  },
  {
    id: 30,
    title: 'Benita',
    size: 20,
  },
  {
    id: 31,
    title: 'Harrington',
    size: 27,
  },
  {
    id: 32,
    title: 'Hobbs',
    size: 17,
  },
  {
    id: 33,
    title: 'Fuentes',
    size: 26,
  },
  {
    id: 34,
    title: 'Alyce',
    size: 22,
  },
  {
    id: 35,
    title: 'Frank',
    size: 27,
  },
  {
    id: 36,
    title: 'Lawrence',
    size: 21,
  },
  {
    id: 37,
    title: 'Harper',
    size: 19,
  },
  {
    id: 38,
    title: 'Woods',
    size: 29,
  },
  {
    id: 39,
    title: 'Hopper',
    size: 25,
  },
  {
    id: 40,
    title: 'Lesley',
    size: 28,
  },
  {
    id: 41,
    title: 'Arlene',
    size: 30,
  },
  {
    id: 42,
    title: 'Mcdowell',
    size: 13,
  },
  {
    id: 43,
    title: 'Kristy',
    size: 12,
  },
  {
    id: 44,
    title: 'Janette',
    size: 21,
  },
  {
    id: 45,
    title: 'Charles',
    size: 20,
  },
  {
    id: 46,
    title: 'Houston',
    size: 30,
  },
  {
    id: 47,
    title: 'Renee',
    size: 32,
  },
  {
    id: 48,
    title: 'Leah',
    size: 16,
  },
  {
    id: 49,
    title: 'Marguerite',
    size: 31,
  },
  {
    id: 50,
    title: 'Moreno',
    size: 18,
  },
  {
    id: 51,
    title: 'Washington',
    size: 21,
  },
  {
    id: 52,
    title: 'Davis',
    size: 14,
  },
  {
    id: 53,
    title: 'Potts',
    size: 29,
  },
  {
    id: 54,
    title: 'Holly',
    size: 22,
  },
  {
    id: 55,
    title: 'Phillips',
    size: 18,
  },
  {
    id: 56,
    title: 'Santiago',
    size: 32,
  },
  {
    id: 57,
    title: 'Suarez',
    size: 24,
  },
  {
    id: 58,
    title: 'Lynnette',
    size: 12,
  },
  {
    id: 59,
    title: 'Gates',
    size: 18,
  },
  {
    id: 60,
    title: 'Aurelia',
    size: 14,
  },
  {
    id: 61,
    title: 'Horn',
    size: 30,
  },
  {
    id: 62,
    title: 'Annabelle',
    size: 19,
  },
  {
    id: 63,
    title: 'Snyder',
    size: 19,
  },
  {
    id: 64,
    title: 'Tanner',
    size: 24,
  },
  {
    id: 65,
    title: 'Walker',
    size: 26,
  },
  {
    id: 66,
    title: 'Ruthie',
    size: 26,
  },
  {
    id: 67,
    title: 'Beverly',
    size: 20,
  },
  {
    id: 68,
    title: 'Ines',
    size: 16,
  },
  {
    id: 69,
    title: 'Dudley',
    size: 17,
  },
  {
    id: 70,
    title: 'Hays',
    size: 19,
  },
  {
    id: 71,
    title: 'Russell',
    size: 32,
  },
  {
    id: 72,
    title: 'Lolita',
    size: 30,
  },
  {
    id: 73,
    title: 'Kasey',
    size: 25,
  },
  {
    id: 74,
    title: 'Abby',
    size: 26,
  },
  {
    id: 75,
    title: 'Mason',
    size: 16,
  },
  {
    id: 76,
    title: 'Wilcox',
    size: 12,
  },
  {
    id: 77,
    title: 'Talley',
    size: 27,
  },
  {
    id: 78,
    title: 'Hull',
    size: 31,
  },
  {
    id: 79,
    title: 'Harrison',
    size: 15,
  },
  {
    id: 80,
    title: 'Cooke',
    size: 23,
  },
  {
    id: 81,
    title: 'Sparks',
    size: 15,
  },
  {
    id: 82,
    title: 'Calhoun',
    size: 23,
  },
  {
    id: 83,
    title: 'Deborah',
    size: 18,
  },
  {
    id: 84,
    title: 'Glass',
    size: 26,
  },
  {
    id: 85,
    title: 'Butler',
    size: 30,
  },
  {
    id: 86,
    title: 'Barton',
    size: 16,
  },
  {
    id: 87,
    title: 'Carpenter',
    size: 19,
  },
  {
    id: 88,
    title: 'Roberta',
    size: 16,
  },
  {
    id: 89,
    title: 'Lester',
    size: 23,
  },
  {
    id: 90,
    title: 'Sonya',
    size: 30,
  },
  {
    id: 91,
    title: 'Newman',
    size: 24,
  },
  {
    id: 92,
    title: 'Barron',
    size: 12,
  },
  {
    id: 93,
    title: 'Jackie',
    size: 18,
  },
  {
    id: 94,
    title: 'Margarita',
    size: 20,
  },
  {
    id: 95,
    title: 'Taylor',
    size: 15,
  },
  {
    id: 96,
    title: 'Rose',
    size: 21,
  },
  {
    id: 97,
    title: 'Hammond',
    size: 29,
  },
  {
    id: 98,
    title: 'Berg',
    size: 27,
  },
  {
    id: 99,
    title: 'Dennis',
    size: 22,
  },
];
export const edges = [
  {
    source: 5,
    target: 6,
    title: 'edge title',
  },
  {
    source: 42,
    target: 89,
  },
  {
    source: 84,
    target: 11,
  },
  {
    source: 60,
    target: 10,
  },
  {
    source: 17,
    target: 74,
  },
  {
    source: 28,
    target: 79,
  },
  {
    source: 37,
    target: 84,
  },
  {
    source: 32,
    target: 46,
  },
  {
    source: 2,
    target: 24,
  },
  {
    source: 33,
    target: 24,
  },
  {
    source: 76,
    target: 78,
  },
  {
    source: 19,
    target: 70,
  },
  {
    source: 45,
    target: 36,
  },
  {
    source: 74,
    target: 87,
  },
  {
    source: 59,
    target: 33,
  },
  {
    source: 14,
    target: 66,
  },
  {
    source: 10,
    target: 35,
  },
  {
    source: 77,
    target: 57,
  },
  {
    source: 13,
    target: 37,
  },
  {
    source: 39,
    target: 18,
  },
  {
    source: 37,
    target: 3,
  },
  {
    source: 14,
    target: 47,
  },
  {
    source: 85,
    target: 20,
  },
  {
    source: 83,
    target: 84,
  },
  {
    source: 65,
    target: 98,
  },
  {
    source: 96,
    target: 62,
  },
  {
    source: 15,
    target: 98,
  },
  {
    source: 89,
    target: 8,
  },
  {
    source: 26,
    target: 49,
  },
  {
    source: 14,
    target: 94,
  },
  {
    source: 47,
    target: 14,
  },
  {
    source: 68,
    target: 84,
  },
  {
    source: 52,
    target: 3,
  },
  {
    source: 66,
    target: 47,
  },
  {
    source: 53,
    target: 16,
  },
  {
    source: 46,
    target: 73,
  },
  {
    source: 99,
    target: 27,
  },
  {
    source: 84,
    target: 10,
  },
  {
    source: 14,
    target: 28,
  },
  {
    source: 74,
    target: 50,
  },
  {
    source: 30,
    target: 37,
  },
  {
    source: 49,
    target: 32,
  },
  {
    source: 5,
    target: 1,
  },
  {
    source: 4,
    target: 61,
  },
  {
    source: 25,
    target: 45,
  },
  {
    source: 29,
    target: 8,
  },
  {
    source: 14,
    target: 76,
  },
  {
    source: 9,
    target: 71,
  },
  {
    source: 58,
    target: 47,
  },
  {
    source: 72,
    target: 95,
  },
  {
    source: 44,
    target: 81,
  },
  {
    source: 33,
    target: 57,
  },
  {
    source: 61,
    target: 87,
  },
  {
    source: 56,
    target: 65,
  },
  {
    source: 80,
    target: 79,
  },
  {
    source: 24,
    target: 54,
  },
  {
    source: 94,
    target: 67,
  },
  {
    source: 5,
    target: 66,
  },
  {
    source: 55,
    target: 85,
  },
  {
    source: 33,
    target: 53,
  },
  {
    source: 32,
    target: 39,
  },
  {
    source: 41,
    target: 20,
  },
  {
    source: 7,
    target: 47,
  },
  {
    source: 71,
    target: 6,
  },
  {
    source: 23,
    target: 61,
  },
  {
    source: 41,
    target: 34,
  },
  {
    source: 5,
    target: 42,
  },
  {
    source: 90,
    target: 72,
  },
  {
    source: 70,
    target: 49,
  },
  {
    source: 99,
    target: 25,
  },
  {
    source: 69,
    target: 46,
  },
  {
    source: 65,
    target: 90,
  },
  {
    source: 93,
    target: 72,
  },
  {
    source: 7,
    target: 26,
  },
  {
    source: 88,
    target: 39,
  },
  {
    source: 77,
    target: 52,
  },
  {
    source: 86,
    target: 38,
  },
  {
    source: 47,
    target: 38,
  },
  {
    source: 67,
    target: 51,
  },
  {
    source: 41,
    target: 12,
  },
  {
    source: 29,
    target: 71,
  },
  {
    source: 86,
    target: 42,
  },
  {
    source: 80,
    target: 89,
  },
  {
    source: 54,
    target: 53,
  },
  {
    source: 93,
    target: 25,
  },
  {
    source: 28,
    target: 72,
  },
  {
    source: 70,
    target: 59,
  },
  {
    source: 18,
    target: 88,
  },
  {
    source: 60,
    target: 4,
  },
  {
    source: 19,
    target: 0,
  },
  {
    source: 78,
    target: 37,
  },
  {
    source: 85,
    target: 54,
  },
  {
    source: 83,
    target: 44,
  },
  {
    source: 84,
    target: 12,
  },
  {
    source: 43,
    target: 70,
  },
  {
    source: 27,
    target: 81,
  },
  {
    source: 85,
    target: 55,
  },
  {
    source: 53,
    target: 69,
  },
  {
    source: 46,
    target: 87,
  },
  {
    source: 80,
    target: 98,
  },
  {
    source: 31,
    target: 10,
  },
  {
    source: 49,
    target: 13,
  },
  {
    source: 61,
    target: 39,
  },
  {
    source: 64,
    target: 10,
  },
  {
    source: 96,
    target: 58,
  },
  {
    source: 43,
    target: 23,
  },
  {
    source: 99,
    target: 53,
  },
  {
    source: 52,
    target: 21,
  },
  {
    source: 12,
    target: 88,
  },
  {
    source: 5,
    target: 11,
  },
  {
    source: 34,
    target: 42,
  },
  {
    source: 31,
    target: 97,
  },
  {
    source: 56,
    target: 49,
  },
  {
    source: 27,
    target: 74,
  },
  {
    source: 11,
    target: 30,
  },
];
