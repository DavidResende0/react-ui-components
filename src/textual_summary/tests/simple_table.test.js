import React from 'react';
import renderer from 'react-test-renderer';
import SimpleTable from '../simple_table';
import { simpleTableData } from '../data/simple_table';

describe('Simple Table', () => {
  it('renders just fine...', () => {
    /*
     * label:
     *   a) simple
     *   b) object {sortable: true} TODO
     * rows
     *   value:
     *     a) simple
     *     b) object {expandable: true} TODO
     */
    const component = renderer.create(<SimpleTable
      title={simpleTableData.title}
      labels={simpleTableData.labels}
      rows={simpleTableData.rows}
      onClick={e => null}
    />);
    const table = component.toJSON();
    expect(table).toMatchSnapshot();
  });
});
