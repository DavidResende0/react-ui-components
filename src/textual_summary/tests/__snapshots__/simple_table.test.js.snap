// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Simple Table renders just fine... 1`] = `
<SimpleTable
  labels={
    Array [
      "Network Protocol",
      "Host Protocol",
      "Direction",
      "Port",
      "End Port",
      "Source",
    ]
  }
  rows={
    Array [
      Array [
        null,
        "-1",
        "outbound",
        0,
        0,
        "0.0.0.0/0",
      ],
      Array [
        null,
        "TCP",
        "inbound",
        22,
        22,
        "0.0.0.0/0",
      ],
      Array [
        null,
        "TCP",
        "inbound",
        80,
        80,
        "0.0.0.0/0",
      ],
    ]
  }
  title="Firewall Rules"
>
  <table
    className="table table-bordered table-striped table-summary-screen"
  >
    <thead>
      <tr>
        <th
          align="left"
          colSpan={6}
        >
          Firewall Rules
        </th>
      </tr>
      <tr>
        <td
          key="0"
        >
          <strong>
            Network Protocol
          </strong>
        </td>
        <td
          key="1"
        >
          <strong>
            Host Protocol
          </strong>
        </td>
        <td
          key="2"
        >
          <strong>
            Direction
          </strong>
        </td>
        <td
          key="3"
        >
          <strong>
            Port
          </strong>
        </td>
        <td
          key="4"
        >
          <strong>
            End Port
          </strong>
        </td>
        <td
          key="5"
        >
          <strong>
            Source
          </strong>
        </td>
      </tr>
    </thead>
    <tbody>
      <tr
        key="0"
      >
        <td
          key="0"
        />
        <td
          key="1"
        >
          -1
        </td>
        <td
          key="2"
        >
          outbound
        </td>
        <td
          key="3"
        >
          0
        </td>
        <td
          key="4"
        >
          0
        </td>
        <td
          key="5"
        >
          0.0.0.0/0
        </td>
      </tr>
      <tr
        key="1"
      >
        <td
          key="0"
        />
        <td
          key="1"
        >
          TCP
        </td>
        <td
          key="2"
        >
          inbound
        </td>
        <td
          key="3"
        >
          22
        </td>
        <td
          key="4"
        >
          22
        </td>
        <td
          key="5"
        >
          0.0.0.0/0
        </td>
      </tr>
      <tr
        key="2"
      >
        <td
          key="0"
        />
        <td
          key="1"
        >
          TCP
        </td>
        <td
          key="2"
        >
          inbound
        </td>
        <td
          key="3"
        >
          80
        </td>
        <td
          key="4"
        >
          80
        </td>
        <td
          key="5"
        >
          0.0.0.0/0
        </td>
      </tr>
    </tbody>
  </table>
</SimpleTable>
`;
